README.md

```diff
@@ -44,20 +44,21 @@ You can participate in the task force work at various levels. In order of increa
 
 このタスクフォースにはいくつかの立場で参加可能です。活動を活発にするために、追跡者、貢献者、参加者、編者、議長の分類があります。それぞれの詳細は、[参加の立場](https://www.w3.org/International/i18n-drafts/pages/task_force_roles)についてのリストを参照してください。
 
-**To just follow the work:** Rather than 'Watch' this repository, [subscribe](mailto:public-i18n-japanese-request@w3.org?subject=subscribe) to the [public-i18n-japanese](https://lists.w3.org/Archives/Public/public-i18n-japanese/) mailing list. That list is notified (no more than once a day, and in digest form), about changes to issues in this repository, but also about other W3C Working Group issues related to the Japanese writing systems.
+**To just follow the work:** Rather than 'Watch' this repository, [subscribe](mailto:public-jlreq-admin-request@w3.org?subject=subscribe) to the [public-jlreq-admin](https://lists.w3.org/Archives/Public/public-jlreq-admin/) mailing list. That list is notified (no more than once a day, and in digest form), about changes to issues in this repository, but also about other W3C Working Group issues related to the Japanese writing systems.
```

Comment:
Why is this changed?  People who want to follow the work (without joining the group) want to follow the technical discussion, not the admin.

Reply:
Sorry, this is draft under discussion with Kida-san.